tasks/winbuildscripts/Generate-Pyzip.ps1

```diff
@@ -55,6 +55,7 @@ if($maj -eq "2") {
     Start-Process msiexec -ArgumentList "/q /i $dlfullpath TARGETDIR=c:\pythonroot ADDLOCAL=DefaultFeature,Tools,TclTk,Testsuite" -Wait
 } 
 elseif ($maj -eq "3") {
+    $Env:PATH="$Env:PATH;c:\tools\msys64\mingw64\bin"
```

Comment:
I believe doing `ridk enable` here would give the same result. I don't know what's better here, so let's go with your solution.

Question: This will also add the x64 toolchain to the path when generating the x86 Python package. Is that on purpose? Otherwise we should only conditionally include this.

Reply:
yeah, I need to go back and try the x86 version :(